components/match2/wikis/counterstrike/match_group_input_custom.lua

```diff
@@ -39,6 +39,11 @@ local CustomMatchGroupInput = {}
 ---@param options table?
 ---@return table
 function CustomMatchGroupInput.processMatch(match, options)
+	match.status = Logic.emptyOr(match.status, Variables.varDefault('tournament_status'), match.finished)
+	if (not Logic.readBool(match.finished)) and Logic.isNotEmpty(match.status) then
+		match.finished = match.status
+	end
+
 	local finishedInput = match.finished --[[@as string?]]
```

Comment:
```suggestion
	local finishedInput = Logic.emptyOr(match.status, Variables.varDefault('tournament_status'), match.finished) --[[@as string?]]
```
Wouldn't this do it?

Preferably with the `match.status` input removed

Reply:
fwiw match.status is needed somewhere down the line
shitty to check on phone

Reply:
Why would match.status be needed?

Reply:
not the same as before, it would ignore bool input to finished if status is set
could do
```lua
	---@type string|boolean?
	local finishedInput = Logic.nilOr(
		Logic.readBoolOrNil(match.finished),
		Logic.nilIfEmpty(match.status),
		Variables.varDefault('tournament_status'),
		match.finished
	)
```

Reply:
Do we need match.status though?